ground/src/main/java/com/google/android/ground/ui/map/MapProviderModule.java

```diff
@@ -16,7 +16,7 @@
 
 package com.google.android.ground.ui.map;
 
-import com.google.android.ground.ui.map.gms.GoogleMapsFragment;
+import com.google.android.ground.ui.map.gms.GoogleMapsFragmentV2;
```

Comment:
Can we remove V2 now?

Reply:
I thought it might be helpful to keep the original GoogleMapsFragment around at least until we finish rendering parity just so we can have the context for how it used to be done. Although I suppose we could also just look in the code history for that.

WDYT?

Reply:
I'm leaning towards having a clean history (one version replaces the other), since the change is breaking anyway, but it's up to you.

Reply:
Sounds good, replaced GoogleMapsFragment with the Kotlin version